types/developerTestAccounts.ts

```diff
@@ -22,6 +22,7 @@ export type CreateDeveloperTestAccountResponse = {
 export type CreateDeveloperTestAccountV3Response = {
   id: number;
   accountName: string;
+  personalAccessKey?: string;
```

Comment:
TS was complaining about this in the build so I fixed. Unrelated to this PR though

Reply:
hmm TS error is now gone after removal, I must have been in a weird state